SourceX/DiabloUI/dialogs.cpp

```diff
@@ -80,62 +79,42 @@ UiItem SELOK_DIALOG[] = {
 };
 
 UiItem SPAWNERR_DIALOG[] = {
-	UiText("The Rogue and Sorcerer are only available in the full retail version of Diablo. For ordering information call (800) 953-SNOW.", { 140, 199, 400, 177 }),
+	UiText("The Rogue and Sorcerer are only available in the full retail version of Diablo. For ordering information visit https://www.gog.com/game/diablo.", { 140, 199, 400, 177 }),
 	UiArtTextButton("OK", &DialogActionOK, { 230, 407, 180, 43 }),
 };
 
-const std::uint_fast8_t DEFAULT_BG_DIALOG_IDX = 4;
-const std::uint_fast8_t DEFAULT_BG_BUTTON_IDX = 5;
-
-void LoadFallbackPalette()
+void StubArt(Art *art, int w, int h, BYTE *artData, int frames)

```

Comment:
This is not really stubbing the art anymore.
Consider renaming to `LoadArt` and moving to `art.h/cpp`.

Reply:
Keep forgetting about function overloading :)